### ISSUE TITLE
Inconsistent command/session

### ISSUE THREAD

**Issue by bertini97** [apognu/tuigreet#139]
Tuigreet displays, under the `CMD:` label, either the session it's about to start or the command to launch. If you manually select a session, for example "Sway", you will get `CMD: Sway`, and then you will login into sway. However, if you ask tuigreet to remember it, at the next login you will be presented with `CMD: sway`, which is the command.

Here session and command are almost the same, but take for example the case of SwayFX, where the session is called "SwayFX" but shares the same command `sway`.

I think if tuigreet last started a session, and not a command, it should display the session name. But I also think a better solution would be to provide the name of the session and the command separately, as to avoid all confusion. Something like
```
ESC  F2: Change command  F3: Change session  SESSION: Sway  COMMAND: sway
ESC  F2: Change command  F3: Change session  SESSION: None  COMMAND: /bin/bash
```

Or even
```
ESC  SESSION: Sway (F3: change)  COMMAND: sway (F2: change)
```

Thanks for the attention.

**Comment by apognu** [comment]
You haven't filled out the template, please provide the required information. For example, which version of `tuigreet` are you using are what is your command-line for running it?

We changed the way sessions are remembered a while back and, save for a bug (which is possible), the actual `Name` of the remembered session should now be displayed, not its command.

Thank you.

**Comment by bertini97** [comment]
Yes, sorry. I'm running `tuigreet 0.9.0`, and the command is
```
tuigreet --remember --remember-session --asterisks
```
So far the only indication that `tuigreet` is showing the command and not the session is because it's not capitalized. I'll test some more and report back.

**Comment by apognu** [comment]
I think I might see where the issue might be. It should only occur with `--remember-session`, not `--remember-user-session` and in this case, it would prefer the command name instead of the session.

I'll confirm when I get a computer back and if it is the case, it is indeed not the intended behavior.

**Comment by apognu** [comment]
I think I found the issue, it will be included in the 0.9.1 hotfix. Thanks for reporting.

**Comment by apognu** [comment]
This is now available in the 0.9.1 version of `tuigreet`.
